src/components/@shared/FormInput/InputElement/ConsumerParameters/_validation.ts

```diff
@@ -34,7 +34,7 @@ export const validationConsumerParameters: {
       )
       return occasions.length === 1
     })
-    .min(4, (param) => `Name must be at least ${param.min} characters`)
+    /* .min(1, (param) => `Name must be at least ${param.min} characters`) */
```

Comment:
question(blocking): why did we remove the line entirely instead of lowering to 1?